src/TestRunnerJSON.jl

```diff
@@ -0,0 +1,38 @@
+using JSON
+
+function json(testStructureNodes::Array{TestRunner.TestStructureNode})
+  treeDictionary = [to_dict(node) for node in testStructureNodes]
+  JSON.json(treeDictionary)
+end
+
+function to_dict(factGroup::FactsNode)
```

Comment:
@mateuszkaleta , can you change it to FactsCollectionNode? That's the new name for FactsNode.